tests/ast/data/windows.test

```diff
@@ -557,6 +557,9 @@ body {
                                 }
                               }
                             }
```

Comment:
Do we have a test for this where it is false?

Reply:
It can't be "false" -- it's an enum. It can only be set to true but this should be tested by the asc/desc tests.